include/tbl/params.h

```diff
@@ -464,6 +464,19 @@ PARAM_SIMPLE(
 	/* flags */	EXPERIMENTAL
 )
 
+PARAM_SIMPLE(
+	/* name */	pass_prefetch,
+	/* type */	bytes,
+	/* min */	"4k",
+	/* max */	NULL,
+	/* def */	"1M",
+	/* units */	"bytes",
+	/* descr */
+	"The default prefetch amount used during a single private transaction.\n"
+	"Enabling this will prevent running out of memory when "
+	"there are big transfers going on in the background."
```

Comment:
this could be confused with bgfetches, I suggest making clear that this applies to streaming passes.